setup.cfg

```diff
@@ -11,3 +11,7 @@ max-complexity = 18
 exclude =
   __init__.py
   napari_arboretum/_tests
+
+[isort]
+profile = black
+line_length = 79
```

Comment:
Currently, this code base is using `black`'s (sensible) default of `88`. Doing this here will mean that `isort` is using `black`'s profile with line length `79` when the rest of the code will be using `88`. IMO, we shouldn't be changing unless there's a good reason. The ability to change to something `!= 88` is for legacy code bases where `black` has been introduced at a much later date, which is not the case here

Reply:
I think I just put this in as a default, and I'm not that bothered about the particlar value. Should I just change this line to 88?

Reply:
I changed this to 88

Reply:
You don't need to. `isort`'s `black` profile will use `black` defaults which is 88